packages/spark/base/inputs/_text-icon-input.scss

```diff
@@ -1,6 +1,6 @@
-.sprk-b-TextInputIconContainer--has-text-icon {
+.sprk-b-TextInputIconContainer--has-text-icon,
+.sprk-b-TextInputIconContainer--has-text-icon-right {
```

Comment:
Does this class need to be added here? We switched to use an svg instead, so it shouldnt need any of the styles on the ::before element, right?